### PR TITLE
Add mini NVS solver example

### PR DESCRIPTION
**Pull request by pablovela5620**
### What

A new example for Novel View Synthesis based on [NVS-Solver](https://github.com/ZHU-Zhiyu/NVS_Solver).

This example demonstrates using rerun to visualize the internal workings of [NVS-Solver](https://github.com/ZHU-Zhiyu/NVS_Solver). With a specific focus on the single image case. Still a work in progress, waiting for 0.17 gradio_rerun component and need a bit more time to convert to a Huggingface Zero Space.

[Link to the external repo](https://github.com/pablovela5620/mini-nvs-solver)

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6888?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6888?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/6888)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by jprochazk** [comment]
@rerun-bot approve

**Review by nikolausWest — APPROVED** [review]
Nice

File: examples/python/mini_nvs_solver/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--[metadata]
+title = "Mini NVS Solver"
```
Comment (nikolausWest) [inline examples/python/mini_nvs_solver/README.md:2]:
```suggestion
title = "Mini NVS solver"
```

File: examples/python/mini_nvs_solver/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!--[metadata]
+title = "Mini NVS solver"
+tags = ["2D", "3D", "HuggingFace", "Depth", "Pinhole camera", "Diffusion"]
+source = "https://github.com/pablovela5620/mini-nvs-solver"
+thumbnail = "https://static.rerun.io/mini-nvs-solver-thumbnail/9a9cadb7a5a3beeabbdc2f4490532c1b24765dd2/480w.png"
+thumbnail_dimensions = [480, 480]
```
Comment (nikolausWest) [inline examples/python/mini_nvs_solver/README.md:6]:
```suggestion
thumbnail_dimensions = [480, 276]
```

**Pull request merged**